### DISCUSSION TITLE
Self-hosted - Restrict registered users from creating projects?

### DISCUSSION THREAD

**Discussion by mojimba** [healthchecks/healthchecks#862] [Q&A]
Hi,

I'm trying out a self-hosted instance of Healthchecks (via docker for now) and I wonder if there's a way to restrict _registered_ users from creating projects? 

What I did so far was: 
1. Using my own admin account, created a few projects, each with some checks and integrations configured
2. Invited users to the project(s), with read-only access
3. Once the invited users have logged in using their magic link and/or created a password, they can see the project(s) they have been invited to, as expected

But... these users can also now create their own projects with separate integrations, checks, etc.

I want to prevent this altogether, is that possible? I have found some user/group permission settings in the backend admin UI, but I'm not entirely sure which permission(s) I should use, and whether these work (for example, adding "accounts | project | Can view project" user permission doesn't seem to have any effect.

Thanks!
-m

**Comment by cuu508** [comment]
No, this is not currently not possible. 

The permissions you see in admin are only used in Django admin. For example, if you remove staff user's "Can view project" permission, they will not be able to view projects in the admin UI.
